poetry/repositories/pypi_repository.py

```diff
@@ -322,8 +322,11 @@ def _get_release_info(self, name: str, version: str) -> dict:
         return data.asdict()
 
     def _get(self, endpoint: str) -> Union[dict, None]:
+        headers = {
```

Comment:
Looks like some unrelated changes came along for the ride. Could we back these out, and open a separate PR with tests if they are necessary?

Reply:
Ah, good catch. I was trying to debug the mac failures from the CI. I'll back these out in a new pull request.